### ISSUE TITLE
Chest Peripheral Wrap Assertion Error

### ISSUE THREAD

**Issue by MamboDancer** [neumond/python-computer-craft#15]
Cant wrap chest peripheral. I get assertion error. Maybe doing something wrong? Thanks!

**Comment by MamboDancer** [comment]
Noticed that github version differs from pip installation.

**Comment by neumond** [comment]
Yep, current trunk isn't well tested yet. What version of MC and CC:T do you use?

**Comment by MamboDancer** [comment]
MC 1.20.1 and CC:T 1.108.3
Also noticed that your py lib which installs with pip doesnt have peripheral initialization of chests furnaces etc, but github version does
Tried to merge them but nothing good happened :)

**Comment by MamboDancer** [comment]
Installed earlier version. Got NoneType error

**Comment by MamboDancer** [comment]
Trying to install module from git repo, got errors on any module usage

**Comment by MamboDancer** [comment]
So i somehow fixed it and now its working with chest peripheral.
